PlayerControls/sources/ContentControlsViewController+Prism.swift

```diff
@@ -0,0 +1,249 @@
+        }
+        set {
+            guard let newValue = newValue else { fatalError("Setting nil value forbidden") }
+            self = .nonplayable(newValue)
+        }
+    }
+}
+
+public extension ContentControlsViewController.Props.Player.Item.Controls.Playback {
+    public var none: Bool {
+        get {
+            guard case .none = self else { return false }
+            return true
+        }
+        set {
+            guard newValue else { fatalError("Setting false value forbidden") }
+            self = .none
+        }
+    }
+    
+    public var play: Action<Void>? {
+        get {
+            guard case let .play(play) = self else { return nil }
+            return play
+        }
+        set {
+            guard let newValue = newValue else { fatalError("Setting nil value forbidden") }
+            self = .play(newValue)
+        }
+    }
+    
+    public var pause: Action<Void>? {
+        get {
+            guard case let .pause(pause) = self else { return nil }
+            return pause
+        }
+        set {
+            guard let newValue = newValue else { fatalError("Setting nil value forbidden") }
+            self = .pause(newValue)
+        }
+    }
+    
+    public var replay: Action<Void>? {
+        get {
+            guard case let .replay(replay) = self else { return nil }
+            return replay
+        }
+        set {
+            guard let newValue = newValue else { fatalError("Setting nil value forbidden") }
+            self = .replay(newValue)
+        }
+    }
+}
+
+public extension ContentControlsViewController.Props.Player.Item.Controls.Subtitles {
+    public var none: Bool {
+        get {
+            guard case .none = self else { return false }
+            return true
+        }
+        set {
+            guard newValue else { fatalError("Setting false value forbidden") }
+            self = .none
+        }
+    }
+    
+    public var unavailable: Bool {
+        get {
+            guard case .unavailable = self else { return false }
+            return true
+        }
+        set {
+            guard newValue else { fatalError("Setting false value forbidden") }
+            self = .unavailable
+        }
+    }
+    
+    public var available: (toggle: Action<Void>, state: State)? {
+        get {
+            guard case let .available(available) = self else { return nil }
+            return available
+        }
+        set {
+            guard let newValue = newValue else { fatalError("Setting nil value forbidden") }
+            self = .available(toggle: newValue.toggle, state: newValue.state)
+        }
+    }
+}
+
+public extension ContentControlsViewController.Props.Player.Item.Controls.Subtitles.State {
+    public var active: String? {
+        get {
+            guard case let .active(active) = self else { return nil }
+            return active
+        }
+        set {
+            guard let newValue = newValue else { fatalError("Setting nil value forbidden") }
```

Comment:
How about make this properties get only? Such way to set var is complicates code reading, i think.

`ContentControlsViewController.props.isNoPlayer = true`
vs
`ContentControlsViewController.props. = .noPlayer`